Modules/_ctypes/_ctypes.c

```diff
@@ -4814,7 +4816,13 @@ PyTypeObject PyCArray_Type = {
     0,                                          /* tp_setattro */
     &PyCData_as_buffer,                         /* tp_as_buffer */
     Py_TPFLAGS_DEFAULT | Py_TPFLAGS_BASETYPE, /* tp_flags */
-    PyDoc_STR("XXX to be provided"),            /* tp_doc */
+    PyDoc_STRVAR(array_doc,
```

Comment:
The error is here. Please take a look at the usage of `PyDoc_STRVAR`, you can search this in the source code of cpython :)

BTW. There is already [a similar PR](https://github.com/python/cpython/issues/107689) here.